spdm.rflx

```diff
@@ -0,0 +1,1487 @@
+      (Digests                   => 16#01#,
+       Certificate               => 16#02#,
+       Challenge_Auth            => 16#03#,
+       Version                   => 16#04#,
+       Measurements              => 16#60#,
+       Capabilities              => 16#61#,
+       Algorithms                => 16#63#,
+       Key_Exchange_RSP          => 16#64#,
+       Finish_RSP                => 16#65#,
+       PSK_Exchange_RSP          => 16#66#,
+       PSK_Finish_RSP            => 16#67#,
+       Heartbeat_ACK             => 16#68#,
+       Key_Update_ACK            => 16#69#,
+       Encapsulated_Request      => 16#6A#,
+       Encapsulated_Response_ACK => 16#6B#,
+       End_Session_ACK           => 16#6C#,
+       Vendor_Defined_Response   => 16#7E#,
+       Error                     => 16#7F#)
+   with Size => 8;
+
+   --  Common types
+
+   type Slot_Present is range 0 .. 1 with Size => 1;
+
+   type Slot_Mask is
+      message
+         Slot_7_Present : Slot_Present;
+         Slot_6_Present : Slot_Present;
+         Slot_5_Present : Slot_Present;
+         Slot_4_Present : Slot_Present;
+         Slot_3_Present : Slot_Present;
+         Slot_2_Present : Slot_Present;
+         Slot_1_Present : Slot_Present;
+         Slot_0_Present : Slot_Present;
+      end message;
+
+   -- 10.2 GET_VERSION request and VERSION response messages
+
+   type Get_Version_Request is
+      message
+         Param_1 : Reserved_8;
+         Param_2 : Reserved_8;
+      end message;
+
+   type Version_Number_Entry is
+      message
+         Alpha : Alpha
+            --  Assume a released version
+            then Update_Version_Number
+               if Alpha <= 1;
+         Update_Version_Number : Update_Version_Number;
+         Minor_Version : Minor_Version;
+         Major_Version : Major_Version;
+      end message;
+
+   type Version_Number_Entries is sequence of Version_Number_Entry;
+
+   type Version_Number_Entry_Count is range 1 .. 2**8 - 1 with Size => 8;
+
+   type Version_Response is
+      message
+         Param_1 : Reserved_8;
+         Param_2 : Reserved_8;
+         Reserved : Reserved_8;
+         Version_Number_Entry_Count : Version_Number_Entry_Count;
+         Version_Number_Entries : Version_Number_Entries
+            --  ISSUE: Componolit/RecordFlux#404
+            --  with Size => Version_Number_Entry_Count * Version_Number_Entry'Size;
+            with Size => Version_Number_Entry_Count * 16;
+      end message;
+
+   -- 10.3 GET_CAPABILITIES request and CAPABILITIES response messages
+
+   type CT_Exponent is range 0 .. 2**8 - 1 with Size => 8;
+   type PSK_Request_Cap is
+      (PSK_Req_Unsupported => 0,
+       PSK_Req_Supported   => 1)
+   with Size => 2;
+
+   --  ISSUE: Componolit/RecordFlux#609
+   --  This message differs between SPDM version 1.0 and 1.1.
+   --  Since the minor version is not accessible from
+   --  within this message one message is needed for each version
+   --  that is selected by a refinement.
```

Comment:
Componolit/RecordFlux#609 is fixed. This actually is part of #92 where the duplicate message should be merged again.

Reply:
Fixed in https://github.com/Componolit/RecordFlux-specifications/pull/93/commits/7c1908a0db7eece9cbbb987fa14b0c9d9e4ec91f.